XRTK-Core/Packages/com.xrtk.core/Providers/InputSystem/Simulation/BaseSimulationDataProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using XRTK.Interfaces.Providers.InputSystem.Simulation;
+using XRTK.Services;
+
+namespace XRTK.Providers.InputSystem.Simulation
+{
+    public class BaseSimulationDataProvider : BaseDataProvider, IMixedRealityInputSimulationDataProvider
```

Comment:
Do we actually need this class?

Or is there future plans to implement specific things in this base simulation data provider?

Reply:
I am expecting that there will be shared code between simulation data providers.

Reply:
I think this comes back to my recent comment on the Camera PR.  Are we possibly over complicating this?

Simulation is purely a way to override control of those things we have (or don't want to) something to control an actor.  In this case, allowing keyboard / mouse / gamepad control of a camera instead of using a headset.

For hands simulation, there is the emulator or the other keyboard / mouse / gamepad input to control hands (or alternatively using a real input like Leap motion).

I am wondering, has the MRTK overcomplicated, what potentially should be a simple solution?

Just a simple thought as we delve further in to this, thoughts @FejZa ? (maybe worth a discussion on Slack)

Reply:
We can always put this class back without much refactoring or breaking changes as well. Could we just nix it for now until we write that shared code?

Reply:
Will do

Reply:
Done